compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/Primitives.hs

```diff
@@ -343,12 +351,18 @@ convertPrim _ "EToAnyTemplate"
     ETmLam (mkVar "template", TCon template) $
     EToAny (TCon template) (EVar $ mkVar "template")
 
-convertPrim _ "EToAnyChoice"
+convertPrim _ "EToAnyTemplateChoice"
     (tProxy :-> choice :-> TAny) =
     ETmLam (mkVar "_", tProxy) $
     ETmLam (mkVar "choice", choice) $
     EToAny choice (EVar $ mkVar "choice")
 
+convertPrim _ "EToAnyInterfaceChoice"
+    (TApp proxy (TCon typeId) :-> choice :-> TAny) =
+    ETmLam (mkVar "_", TApp proxy (TCon typeId)) $
```

Comment:
```suggestion
    (tProxy@(TApp _ (TCon typeId)) :-> choice :-> TAny) =
    ETmLam (mkVar "_", tProxy) $
```

Reply:
I know. ;) 

I just followed the preexisting binding for  `"EToAnyContractKey"`.

Reply:
I changed it everywhere (x5)

Reply:
ah that's fair, I've probably done it myself in the past from copying older code and only noticed now. Thanks for changing the rest!